modules/ROOT/pages/index-self-managed.adoc

```diff
@@ -1,26 +1,76 @@
+
+=== Additional Cluster Components
+
+The following Runtime Fabric Components also run inside the Kubernetes cluster: 
+
+* Mule Cluster IP Service: Provides an API that Mule applications use to discover their peers inside the cluster
+* Resource cache: Provides a cluster-local cache of application dependencies
+* Persistence Gateway: Provides a persistent ObjectStore interface to Mule applications
+
+=== About Component Installations and Upgrades
+
+When you install Runtime Fabric on your Kubernetes cluster, the Runtime Fabric command-line utlity (`rtfctl`) swaps an activation data token for a set of activation properties. Properties include credentials to access the Runtime Fabric Docker Registry and credentials to establish an mTLS connection to the Anypoint platform. It then invokes Helm to perform the installation. 
+
+You trigger upgrades to the Runtime Fabric components in your cluster from the Anypoint control plane, specifically from Runtime Manager. When you trigger an upgrade, an upgrade message is processed inside the target cluster, and `rtfctl` invokes Helm to upgrade the components' software versions. If you encounter a problem during an upgrade, you can use `rtfctl` to invoke an Helm rollback and restore the cluster to a stable state. A rollback creates a report in the control plane.
+
+== How Application Deployments Work in Runtime Fabric on Self-Managed Kubernetes 
+
+When you deploy an application in Runtime Fabric on Self-Managed Kubernetes, the following occurs:
+
+. You use Runtime Manager to trigger the application deployment.
+. The Runtime Fabric agent in the target cluster receives a request to deploy the application.
+. Runtime Fabric assigns the application's namespace or creates a new namespace if necessary.
+. Runtime Fabric generates the appropriate Kubernetes resources, including a deployment, ConfigMap, secrets, services, and ingress.
+. The Kubernetes deployment resource may include an init container, which fetches dependencies from the Runtime Fabric resource cache.
+. If the resource cache doesn’t contain required dependencies, Runtime Fabric fetches them from the control plane and adds them to the resource cache.
+
+//image 
+
+=== Assigning Namespaces in Runtime Fabric
+
+Each application is deployed into a Kubernetes namespace based on the application’s environment. 
+
+. Runtime Fabric looks for a namespace with the label `rtf.mulesoft.com/envId=<ANYPOINT_ENVIRONMENT_ID>`. 
+. If Runtime Fabric can't find that label, it looks for a namespace with the name `<ANYPOINT_ENVIRONMENT_ID>`. 
+. If Runtime Fabric can't find that namespace, it creates a new namespace called `<ANYPOINT_ENVIRONMENT_ID>`.
```

Comment:
ditto: since I can't see the images, I can't tell for sure...which probably means that we need to specify this as an example in which the searched-for names are the EKS, AKS, or GKS names...?
Just ignore me if this is just Kim being confused and not possibly your audience!